api/v1alpha1/openapi.yaml

```diff
@@ -1087,6 +1087,163 @@ paths:
         "404":
           description: NotFound
           content: {}
+  /api/v1/templateversions:
+    get:
+      tags:
+        - templateversion
+      description: list template versions
+      operationId: listTemplateVersions
+      parameters:
+        - name: continue
+          in: query
+          description: An optional parameter to query more results from the server. The value of the paramter must match the value of the 'continue' field in the previous list response.
+          required: false
+          schema:
+            type: string
+        - name: labelSelector
+          in: query
+          description: A selector to restrict the list of returned objects by their labels. Defaults to everything.
+          schema:
+            type: string
+        - name: limit
+          in: query
+          description: The maximum number of results returned in the list response. The server will set the 'continue' field in the list response if more results exist. The continue value may then be specified as parameter in a subesquent query.
```

Comment:
s/subesquent/subsequent/

Reply:
Changed all 6 occurrences of this copy-paste typo :)